test/middleware/csp_test.js

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable */
+const sinon = require('sinon')
+const { expect } = require('chai')
+
+const mockRequest = {
+  method: 'GET',
+  body: {},
+  route: { methods: { get: true } },
+  headers: {}
+}
+
+const requireHelper = function requireHelper (module) {
+  delete require.cache[require.resolve(module)]
+  return require(module)
+}
+
+describe('CSP middleware', () => {
+  it('should not apply rules if the feature is switched off', () => {
```

Comment:
Everywhere in this test, “apply rules” could become “send the Content-Security-Policy header” — forgive my being pernickety but I think it’s clearer to talk in terms of the header rather than rules